packages/web/app/features/ProgramRegistry/PatientProgramRegistryUpdateFormModal.jsx

```diff
@@ -1,511 +0,0 @@
-                            'Select a related condition to record category',
-                          )
-                        : null
-                    }
-                    required
-                  />
-                );
-              },
-            },
-            {
-              key: 'reasonForChange',
-              title: (
-                <span id="condition-category-change-reason-label">
-                  <TranslatedText
-                    stringId="programRegistry.updateConditionModal.reasonForChange"
-                    fallback="Reason for change (if applicable)"
-                  />
-                </span>
-              ),
-              width: 250,
-              accessor: (row, groupName, index) => {
-                // Check for date as a proxy for whether the row is new
-                const initialValue = initialValues.conditions[groupName][index]?.date;
-                if (!initialValue) {
-                  return null;
-                }
-                return (
-                  <Field
-                    name={`conditions[${groupName}][${index}].reasonForChange`}
-                    ariaLabelledBy="condition-category-change-reason-label"
-                    component={StyledTextField}
-                    required
-                    disabled={
-                      values.conditions[groupName][index].conditionCategory ===
-                      initialValues.conditions[groupName][index].conditionCategory
-                    }
-                  />
-                );
-              },
-            },
-            {
-              key: 'history',
-              width: 100,
-              accessor: (row, groupName, index) => {
-                // Check for date as a proxy for whether the row is new
-                const initialValue = initialValues.conditions[groupName][index]?.date;
-                if (!initialValue) {
-                  return null;
-                }
-                return (
-                  <ViewHistoryButton>
-                    <TranslatedText
-                      stringId="programRegistry.viewHistory"
-                      fallback="View history"
-                    />
-                  </ViewHistoryButton>
-                );
-              },
-            },
-          ];
-
-          return (
-            <>
-              <Field
-                name="clinicalStatusId"
-                label={<TranslatedText stringId="general.status.label" fallback="Status" />}
-                component={StyledAutocompleteField}
-                suggester={programRegistryStatusSuggester}
-              />
-              <Divider />
-              <Heading5 mt={0} mb={1}>
-                <TranslatedText
-                  stringId="programRegistry.relatedConditions"
-                  fallback="Related conditions"
-                />
-              </Heading5>
-              <StyledFormTable columns={columns} data={groupedData} />
```

Comment:
This form table and all the columns are split out into the basis for RelatedConditionsForm